### PR TITLE
Replace static configuration

### PR DESCRIPTION
**Pull request by zhxymh**
* [ ] Replace all config and remove AElf.Configuration project
   * [x] Network config
   * [ ] Consensus config
   * [x] Runner config
   * [ ] Actor config
   * [x] Parallel config
   * [ ] GrpcLocal config
   * [ ] GrpcRemote config
   * [x] ApiKey config
   * [x] Management config
   * [x] ServiceUrl config
   * [x] MonitorDatabase config
   * [ ] Miners config
   * [ ] Node config
   * [x] TransactionPool config
   * [x] Rpc config
   * [x] Database config
   * [ ] Chain config
* [ ] Remove CommandLineParser

### CODE REVIEW

**Comment by loning** [comment]
IAccountService commits should not appear here

File: AElf.Management.Tests/SideChainServiceTests.cs [hunk 1]
```diff
@@ -25,15 +26,17 @@ public async Task DeployTest()
             arg.LauncherArg = new DeployLauncherArg();
             arg.LauncherArg.IsConsensusInfoGenerator = true;
 
-            var service = new SideChainService();
+            var options = Options.Create(new ManagementOptions());
+            var service = new SideChainService(options);
```
Comment (loning) [inline AElf.Management.Tests/SideChainServiceTests.cs]: if unit tests use new operation, you should make it use IOC container.
if you have any problems, you can leave this module in static configuration, and make other static configuration convert to Options APSP.

File: AElf.Management/Helper/ServiceUrlHelper.cs [hunk 2]
```diff
@@ -1,81 +1,82 @@
...
 using System.Linq;
 using AElf.Common.Application;
 using AElf.Configuration;
-using AElf.Configuration.Config.Management;
 using k8s;
 
 namespace AElf.Management.Helper
 {
-    public class ServiceUrlHelper
+    public class ServiceUrlHelper0
```
Comment (loning) [inline AElf.Management/Helper/ServiceUrlHelper.cs]: ServiceUrlHelper**0**

File: AElf.Management/Services/AkkaService.cs [hunk 3]
```diff
@@ -1,21 +1,28 @@
...
 using AElf.Management.Request;
+using Microsoft.Extensions.Options;
 
 namespace AElf.Management.Services
 {
     public class AkkaService : IAkkaService
     {
+        private readonly ManagementOptions _managementOptions;
+
+        public AkkaService(IOptions<ManagementOptions> options)
```
Comment (loning) [inline AElf.Management/Services/AkkaService.cs]: IOptionsSnapshot< > * 2

**Comment by loning** [comment]
we should change Management Site into a Abp web startup project like AElf.FullNode. but maybe not in this PR

**Comment by zhxymh** [comment]
> we should change Management Site into a Abp web startup project like AElf.FullNode. but maybe not in this PR

Ok

File: AElf.Core/Modularity/AElfModule.cs [hunk 4]
```diff
@@ -5,6 +5,12 @@ namespace AElf.Modularity
 {
     public abstract class AElfModule : AbpModule
     {
+        protected void ConfigureSelf<TOptions>() where TOptions : class
+        {
```
[the remark this answers is not in the document]
Comment (zhxymh) [reply AElf.Core/Modularity/AElfModule.cs]: Fixed.

**Comment by zhxymh** [comment]
1. Multi-module reference problem :  Consensus, Miners, Node config
2. Use the automatic update feature of the configuration: GrpcLocal,GrpcRemote config
3. Need to use abp bootstrap problem :Actor config

**Comment by loning** [comment]
move unfinished sub issue to a new issue, then we merge

**Comment by zhxymh** [comment]
> move unfinished sub issue to a new issue, then we merge

ok

**Pull request merged**